### PR TITLE
[SMALLFIX] Added documentation about the parameters of the 'checkFilePersisted' method in the 'TfsShellTest' class

### PR DESCRIPTION
**Pull request by HorizonNet**
Add the documentation for the missing parameters of the ```checkFilePersisted``` test.

### CODE REVIEW

**Comment by yupeng9** [comment]
Thanks for the doc fix!

**Comment by yupeng9** [comment]
LGTM

File: shell/src/test/java/tachyon/shell/TfsShellTest.java [hunk 1]
```diff
@@ -1113,7 +1113,10 @@ public void copyToLocalWildcardHierTest() throws IOException, TachyonException {
    * Checks whether the given file is actually persisted by freeing it, then
    * reading it and comparing it against the expected byte array.
    *
-   * @param file
+   * @param file The file to persist
+   * @param size The size of the file
+   * @throws TachyonException
+   * @throws IOException
```
Comment (calvinjia) [inline shell/src/test/java/tachyon/shell/TfsShellTest.java]: Could you complete the javadoc for the exceptions?

Comment (HorizonNet) [inline shell/src/test/java/tachyon/shell/TfsShellTest.java]: Done with the latest update.

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java [hunk 2]
```diff
@@ -1104,7 +1104,10 @@ public void copyToLocalWildcardHierTest() throws IOException, TachyonException {
    * Checks whether the given file is actually persisted by freeing it, then
    * reading it and comparing it against the expected byte array.
    *
-   * @param file
+   * @param file The file to persist
+   * @param size The size of the file
+   * @throws TachyonException Thrown when the file cannot be freed
+   * @throws IOException Thrown either when the file cannot be freed or read
```
Comment (calvinjia) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: This seems to indicate when the file cannot be freed either of the two can be thrown? The current distinction between TachyonException and IOException is if the exception is thrown by a call within Tachyon a TachyonException is thrown. If an exception is thrown by the underlying storage/IO attempt, an IOException will be thrown.

Comment (HorizonNet) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: @calvinjia Thanks for the comment. I updated the PR accordingly.

**Comment by calvinjia** [comment]
LGTM, thanks.

**Pull request merged**